testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -151,20 +153,28 @@ def test_cosine_content():
 
 def test_mean_shape(pca_aligned, u):
     atoms = u.select_atoms(SELECTION)
-    assert_equal(pca_aligned.mean.shape[0], atoms.n_atoms * 3)
+    assert_equal(pca_aligned.mean.shape[0], atoms.n_atoms)
+    assert_equal(pca_aligned.mean.shape[1], 3)
 
 
 def test_calculate_mean(pca_aligned, u, u_aligned):
     ag = u_aligned.select_atoms(SELECTION)
     coords = u_aligned.trajectory.coordinate_array[:, ag.ix]
     assert_almost_equal(pca_aligned.mean, coords.mean(
-        axis=0).ravel(), decimal=5)
+        axis=0), decimal=5)
 
 
-def test_given_mean(pca_aligned, u):
+def test_given_mean(pca, u):
     pca = PCA(u, select=SELECTION, align=False,
-              mean=pca_aligned._mean).run()
-    assert_almost_equal(pca_aligned.cov, pca.cov, decimal=5)
+              mean=pca.mean).run()
+    assert_almost_equal(pca.cov, pca.cov, decimal=5)
+
+
+def test_wrong_num_given_mean(u):
+    wrong_mean = [[0,0,0], [1,1,1]]
```

Comment:
```suggestion
    wrong_mean = [[0, 0, 0], [1, 1, 1]]
```
One does not simply PEP8 into M(ordor)DA?